src/caselawclient/Client.py

```diff
@@ -182,22 +192,17 @@ def POST(
     def get_judgment_xml(
         self, judgment_uri, version_uri=None, show_unpublished=False
     ) -> str:
-        uri = f"/{judgment_uri.lstrip('/')}.xml"
+        uri = self._format_uri(judgment_uri)
```

Comment:
Not one to fix now, necessarily, but I'm struck by how `_format_uri` says nothing about the target format. Should probably contain `xml` in the name somewhere